model/import/Parser/CsvParser.php

```diff
@@ -45,45 +49,34 @@ public function parseFile(File $file, TemplateInterface $template): array
         $lineValidator = $this->getLineValidator();
 
         $items = [];
-
-        foreach ($lines as $line) {
-            $parsedLine = $this->convertCsvLineToArray($line);
-
-            $lineValidator->validate($parsedLine, $template);
-
-            $items[] = $this->convertLineToItem($parsedLine, $template);
+        $validationReport = [];
```

Comment:
```suggestion
        $warningReport = [];
```
Maybe another name to tell, that this is a warning or recoverable.